components/symbl_ai/actions/post-formatted-transcript/post-formatted-transcript.mjs

```diff
@@ -1,11 +1,10 @@
 import symblAIApp from "../../symbl_ai.app.mjs";
-import constants from "../constants.mjs";
 
 export default {
   key: "symbl_ai-post-formatted-transcript",
   name: "Create Formatted Transcript",
   description: "Create formatted transcript from the Conversation. See the doc [here](https://docs.symbl.ai/docs/conversation-api/transcript/).",
-  version: "0.0.1",
+  version: "0.0.8",
```

Comment:
I guess this is suppose to be `0.0.2`

Reply:
It's actually 0.0.1. Thanks for pointing that out. Fixed in 81b88125.